### DISCUSSION TITLE
AAIDrive not working anymore since Update

### DISCUSSION THREAD

**Discussion by Cross199104** [BimmerGestalt/AAIdrive#530] [Q&A] [answered]
Hello together,
I have the situation, that AAIDrive is not working anymore since the latest update.
I plug in the USB cable and MyBMW opens autimatically. Also AAIDrive starts in the background, as soon as the phone connects to the car over BT, but has this strange behaviour.

It connects and disconnects. USB is connected the whole time. The Music Apps are not loaded in the car.

You can see it in the video:

https://user-images.githubusercontent.com/95443362/154406007-b13a40d8-d26b-4c70-87da-29b1f4f0caaf.mp4

I have an Galaxy s10e.
My BMW 1.7.4
AAIDrive 1.3.2-34-bd04b8e
Car is a BMW F11 from 2013 with ID4.

I had BMW Connected 3.1.0.3071 before, but here the same behaviour.

Power restrictions are turned off for both apps.
Phone connected over USB an BT

Is this something known, or am I making a mistake?
Thanks for your support and the great project!

**Comment by hufman** [comment, marked as answer]
Thank you for the video of the problem, including the BCL debug statistics from MyBMW! That is very helpful. The BCL statistics show that no data is being read over the tunnel, and the watchdog timer keeps going up (which is because no ping response is coming back from the car) until it times out and closes the tunnel. This points to some bug in MyBMW itself, because it runs the watchdog timer and is the piece that is shutting down the tunnel.

It sounds a lot like a similar problem that is being tracked down in #510, where USB connections seem to work better if the cable is connected before starting the car. Some of them describe a similar repeated connection/disconnection. Unfortunately I was not able to figure out a reliable solution for USB connections in that thread. Perhaps some of their tips might help you too?

**Reply by Cross199104** [reply]
Hi Hufman,
thank you very much for your help.
I'll try it today after work.

**Reply by Cross199104** [reply]
Hi Hufman.
I tried it today and it works. I have to connect first the phone with the car over USB. I read the #510 and it looks like an issue with cars who have ID4(++) Head Unit.

Thans for your help @hufman
